scripts/wallet-sanity.sh

```diff
@@ -33,6 +33,24 @@ pay_and_confirm() {
   $solana_wallet "${entrypoint[@]}" confirm "$signature"
 }
 
+leader_readiness=false
+timeout=60
+while [[ $timeout -gt 0 ]]; do
+  expected_output="Leader ready"
+  exec 42>&1
+  output=$($solana_wallet "${entrypoint[@]}" get-transaction-count | awk '{print $NF}' | tee >(cat - >&42))
```

Comment:
This is `awk`ward, why not have `solana_wallet get-transaction-count` just output a simple integer?